### PR TITLE
Jammy support

### PR DESCRIPTION
**Pull request by ksimuk**

### CODE REVIEW

**Review by AndreyNikitin — APPROVED** [review]

File: package_list.json [hunk 1]
```diff
@@ -5,161 +5,161 @@
       "xserver": {
         "name": "X server",
         "description": "Virtual X Server and vnc client to connect to. Allows to start GUI apps in the box",
-        "script": "xserver/install.sh",
+        "script": "xserver python3",
```
Comment (vvlevykin) [inline package_list.json:8]: is this intentional or typo?

File: mysql/install.sh [hunk 2]
```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
-
-bash -c "$(curl -fsSL https://raw.github.com/codio/install_software/master/tools/ansible.sh)" mysql
-
-echo "Mysql password root user password is 'codio'"
```
Comment (vvlevykin) [inline mysql/install.sh:5]: Is this needed?

File: mariadb/install.sh [hunk 3]
```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
-
-bash -c "$(curl -fsSL https://raw.github.com/codio/install_software/master/tools/ansible.sh)" mariadb
-
-echo "Mysql password root user password is 'codio'"
```
Comment (vvlevykin) [inline mariadb/install.sh:5]: Is this needed?

File: ungit/install.sh [hunk 4]
```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-
-bash -c "$(curl -fsSL https://raw.github.com/codio/install_software/master/tools/ansible.sh)" ungit
-
-echo
-echo "Add the line to you preview configuration to access ungit from the Menu"
-echo '   "ungit": "https://{{domain8000}}/#/repository?path=%2Fhome%2Fcodio%2Fworkspace"'
```
[the remark this answers is not in the document]
Comment (ksimuk) [reply ungit/install.sh:7]: fixed

File: mariadb/install.sh (hunk 3 above)
Comment (ksimuk) [reply mariadb/install.sh:5]: fixed

**Pull request merged**